x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -812,6 +812,8 @@
     "data.fieldFormats.duration.inputFormats.weeks": "週間",
     "data.fieldFormats.duration.inputFormats.years": "年",
     "data.fieldFormats.duration.negativeLabel": "マイナス",
+    "data.fieldFormats.duration.outputFormats.humanize.approximate": "人間に読解可能 (近似)",
```

Comment:
Could you please undo the changes to translations? 🙏 

We don't contribute to these and those will be re-translated before the release 
https://www.elastic.co/guide/en/kibana/current/contributing.html#kibana-localization

Reply:
I understand. I'll undo these changes.
Should I undo the [change using moment's i18n](https://github.com/elastic/kibana/pull/101479/commits/17b5d4876a540395bb4fdb3c7b948399bfb9e724)?

Reply:
Yes, let's also undo those. 

Otherwise, we should try to be consistent and then need to also change the `human-readable (approximate)` path. I can imagine there are other places we need to change. 

I'd better look into this in a separate issue and pr.

Reply:
Thank you for the advice.
I revered it.